packages/batch/tests/unit/BatchProcessorSync.test.ts

```diff
@@ -41,7 +36,7 @@ describe('Class: BatchProcessor', () => {
   });
 
   describe('Synchronously processing SQS Records', () => {
-    test('Batch processing SQS records with no failures', () => {
+    it('Batch processing SQS records with no failures', () => {
```

Comment:
In Async test you are using a [different](https://github.com/aws-powertools/powertools-lambda-typescript/pull/2938/files#diff-6a137577a0fd6620a72e6cdecea0acd5d9f5f1a21cf36d80970236d22180f6efR39) description. Make sense make them consistent and use the same description?

Reply:
Good catch - I must have forgotten to change these.

I've made the same as the sync one.